requirements/prod.txt

```diff
@@ -1,3 +1,5 @@
 -r base.txt
 gunicorn==20.1.0
 psycopg2==2.9.6
+redis==5.0.0
```

Comment:
sorry, as we discussed this should either go back to `base.txt` or the `CELERY_` settings need to move from `base.py` to `prod.py`

Reply:
done